### PR TITLE
Add missing test for rope_rename.py

### PR DESCRIPTION
**Pull request by benmezger**
This adds a missing test to `plugins/rope_rename.py` improving coverage.

### CODE REVIEW

File: test/plugins/test_rope_rename.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+class Test2(Test1):
+    pass
+"""
+
+
+@pytest.fixture
+def tmp_workspace(workspace):
+    def create_file(name, content):
+        fn = os.path.join(workspace.root_path, name)
```
Comment (ccordoba12) [inline test/plugins/test_rope_rename.py:20]: Where is the `root_path` of this workspace?

Comment (benmezger) [reply test/plugins/test_rope_rename.py:20]: Hey @ccordoba12 sorry for the late reply, I didn't get notified about this comment.
`root_path` is currently set to `/private/var/folders/25/cy4pgsqn2393lvfmh420gf7h0000gn/T/pytest-of-benmezger/pytest-0/test_rope_rename0` on my machine.

Comment (benmezger) [reply test/plugins/test_rope_rename.py:20]: This is actually copied from: https://github.com/palantir/python-language-server/blob/2796154937df0d6552572c6704ff9887e2aa7b33/test/plugins/test_references.py#L26

Comment (ccordoba12) [reply test/plugins/test_rope_rename.py:20]: Ok, thanks for the clarification :+1:

**Review by ccordoba12 — APPROVED** [review]
Thanks @benmezger!

**Pull request merged**